tools/autograd/gen_annotated_fn_args.py

```diff
@@ -39,6 +39,8 @@
 
 
 if TYPE_CHECKING:
+    from collections.abc import Sequence
```

Comment:
To be honest, it's really unclear to me why this is an improvement

Reply:
I also don't know why python introduced this flag...